subsys/bluetooth/shell/bt.c

```diff
@@ -731,6 +731,11 @@ static int cmd_init(const struct shell *sh, size_t argc, char *argv[])
 	return err;
 }
 
+static int cmd_disable(const struct shell *sh, size_t argc, char *argv[])
+{
+	return bt_disable();
```

Comment:
It's not correct to pass on the return value. Shell handler functions pick return values from the following table:

https://github.com/zephyrproject-rtos/zephyr/blob/9331591ab1f6a59513ded32d877f62dbf4526b9e/include/zephyr/shell/shell.h#L134-L137

Reply:
I just realized it's not clear what to return when the command is understood, but not successful. @nordic-krch Do you have a suggestion?